src/Client/ConfigWindow.java

```diff
@@ -1620,74 +1795,92 @@ public void stateChanged(ChangeEvent e) {
         addCheckbox("Remove Report Abuse Button (Similar to prior to 2002-09-11)", overlayPanel);
     overlayPanelRemoveReportAbuseButtonHbarCheckbox.setToolTipText(
         "mudclient149 added the Report Abuse button. You will still be able to report players with right click menu if this option is enabled.");
+    configItems.add(overlayPanelRemoveReportAbuseButtonHbarCheckbox, ConfigWindowTabs.OVERLAYS);
 
     overlayPanelPositionCheckbox = addCheckbox("Display position", overlayPanel);
     overlayPanelPositionCheckbox.setToolTipText("Shows the player's global position");
+    configItems.add(overlayPanelPositionCheckbox, ConfigWindowTabs.OVERLAYS);
 
     overlayPanelRetroFpsCheckbox = addCheckbox("Display FPS like early RSC", overlayPanel);
     overlayPanelRetroFpsCheckbox.setToolTipText(
         "Shows the FPS like it used to be displayed in RSC");
+    configItems.add(overlayPanelRetroFpsCheckbox, ConfigWindowTabs.OVERLAYS);
 
     overlayPanelShowCombatInfoCheckbox = addCheckbox("Show NPC HP info", overlayPanel);
     overlayPanelShowCombatInfoCheckbox.setToolTipText(
         "Shows the HP info for the NPC you're in combat with");
+    configItems.add(overlayPanelShowCombatInfoCheckbox, ConfigWindowTabs.OVERLAYS);
 
     overlayPanelUsePercentageCheckbox = addCheckbox("Use percentage for NPC HP info", overlayPanel);
     overlayPanelUsePercentageCheckbox.setToolTipText(
         "Uses percentage for NPC HP info instead of actual HP");
+    configItems.add(overlayPanelUsePercentageCheckbox, ConfigWindowTabs.OVERLAYS);
 
     overlayPanelLagIndicatorCheckbox = addCheckbox("Lag indicator", overlayPanel);
     overlayPanelLagIndicatorCheckbox.setToolTipText(
         "When there's a problem with your connection, rscplus will tell you in the bottom right");
+    configItems.add(overlayPanelLagIndicatorCheckbox, ConfigWindowTabs.OVERLAYS);
 
     overlayPanelFoodHealingCheckbox =
         addCheckbox("Show food healing overlay (Not implemented yet)", overlayPanel);
     overlayPanelFoodHealingCheckbox.setToolTipText(
         "When hovering on food, shows the HP a consumable recovers");
     // TODO: Remove this line when food healing overlay is implemented
     overlayPanelFoodHealingCheckbox.setEnabled(false);
+    // TODO: Uncomment this line when food healing overlay is implemented
+    // configItems.add(overlayPanelFoodHealingCheckbox, ConfigWindowTabs.OVERLAYS);
 
     overlayPanelHPRegenTimerCheckbox =
         addCheckbox("Display time until next HP regeneration (Not implemented yet)", overlayPanel);
     overlayPanelHPRegenTimerCheckbox.setToolTipText(
         "Shows the seconds until your HP will naturally regenerate");
     // TODO: Remove this line when the HP regen timer is implemented
     overlayPanelHPRegenTimerCheckbox.setEnabled(false);
+    // TODO: Uncomment this line when the HP regen timer is implemented
+    // configItems.add(overlayPanelHPRegenTimerCheckbox, ConfigWindowTabs.OVERLAYS);
```

Comment:
Go ahead and uncomment these now. It seems that not having them in configItems leads to a buggy behaviour. Try typing "NPC" in search (many strings will cause this behaviour), and these disabled checkboxes will remain in the search results, due to them not being managed by the system I guess (Haven't looked into the reason)

Reply:
The search uses `setEnabled` to enable/disable matched items. Since these always want to be disabled, if they were indexed by the search then whenever you clear your search term, these would then become enabled and would let people use them.

The downside is that they'll never be hidden by searching, but since you can never select them anyway, I figured it was a small sacrifice.

The other alternative would be to entirely comment the settings out and only un-comment the entire thing once it's been implemented (which is probably the best fix anyway, rather than showing users non-functional settings).

Reply:
I would like disabled elements to function with the search feature. You can use (and are using, I think) `setVisible` for those elements.